entity-framework/core/extensions/llbl-gen-pro.md

```diff
@@ -0,0 +1,23 @@
+---
+title: LLBLGen Pro | Microsoft Docs
+author: rowanmiller
+ms.author: rowmil
+ms.date: 10/27/2016
+ms.assetid: d66b9e4d-1ed9-4ac5-a473-957a7d82ea92
+ms.technology: entity-framework-core
+uid: core/extensions/llbl-gen-pro
+---
+
+# LLBLGen Pro
+
+> [!NOTE]
+> This documentation is for EF Core. For EF6.x, see [Entity Framework 6](../../ef6/index.md).
+
+> [!NOTE]
+> This extension is not maintained as part of the Entity Framework Core project. When considering a third party extension, be sure to evaluate quality, licensing, support, etc. to ensure they meet your requirements.
+
+LLBLGen Pro is an entity modeling solution for Entity Framework and Entity Framework Core. It lets you easily define your entity model and map it to your database, using database first or model first, so you can get started writing queries right away.
```

Comment:
Should we use "entity modeling solution with support for"? It seems that other O/RMs as well.